trpcultivate_phenotypes/src/TripalCultivatePhenotypesValidatorBase.php

```
@@ -79,14 +79,59 @@ public function getValidatorName() {
 
   /**
    * Get validator plugin validator_scope definition annotation value.
-   * 
+   *
    * @return string
    *   The validator plugin scope annotation definition value.
    */
   public function getValidatorScope() {
     return $this->pluginDefinition['validator_scope'];
   }
 
+  /**
+   * {@inheritdoc}
+   */
+  public function validateRow($row_values, $context) {
```

This should throw an exception since it's a developer caused issue.

NOTE: No need to change this as it will be changed in issue #92 😜